entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# Build application injecting environment variables
+cd /app && npm run build
+
+# Run the serve command
+exec serve -s /app/dist -l 3000
```

Comment:
`npm run build` requires the env variables to be present when we run the command. This was not happening when we run this command in the Dockerfile as part of a RUN clause. By running it in `entrypoint.sh`, the env variables are present in runtime if we pass them through the `docker-compose.yml` file or with `docker run --env-file .env okta-linking-app-frontend`. This is happening because `npm run build` injects the env variable values on the static page it generates.

Reply:
Much better thank you. Now the values passed from `docker run` will be inherited by the `npm run build` which will occur at runtime, instead of occurring at container build time.